### ISSUE TITLE
UniqueMulti rule does not report the error correctly

### ISSUE THREAD

**Issue by axos88-da** [digital-preservation/csv-validator#300]
It reports `unique(` without listing the columns.

```
col1: regex("\d{1,35}") and unique($col1,$col2)
```

reports the error as: ```regex("\d{1,35}") and unique(```

**Comment by DavidUnderdown** [comment]
This actually appears to be a duplicate of #84
